dallinger/experiment_server/experiment_server.py

```diff
@@ -95,21 +96,26 @@ def success_response(**data):
 
 
 def error_response(error_type="Internal server error",
-                   error_text=None,
+                   error_text='',
                    status=400,
-                   participant=None):
+                   participant=None,
+                   simple=False):
     """Return a generic server error response."""
-    traceback.print_exc()
+    if sys.exc_info()[0]:
+        db.logger.exception("Failure for request: {}".format(dict(request.args)))
+        traceback.print_exc()
 
-    page = error_page(
-        error_text=error_text,
-        error_type=error_type,
-        participant=participant)
+    data = {"status": "error"}
+
+    if simple:
+        data["message"] = error_text
+    else:
+        data["html"] = unicode(
```

Comment:
I'm aware that this is just a refactor of old code, but can we fix up the cast to unicode while we're at it?